test/mpi.py

```diff
@@ -253,6 +260,10 @@ def test_set_mpiexec_global_options(self):
     def test_set_mpiexec_opts_from_env(self):
         """test if mpiexec_opts_from_env only contains environment variables that start with the given prefix"""
         mpi_instance = getinstance(mpim.MPI, Local, MympirunOption())
+        check_keys = ['PATH', 'PYTHONPATH']
+        for key in check_keys:
+            if not os.environ.has_key(key):
+                os.environ[key] = "fake %s" % key
```

Comment:
at least make the value look like a `(PYTHON)PATH` value... `/example:/test/123`